### PR TITLE
Bodybuilders can see each others' power levels.

### PR DESCRIPTION
**Pull request by Jacquerel**
## About The Pull Request

Once you gain a level in Athletics you will gain the ability to evaluate the relative muscularity (in terms of unarmed combat) of your fellow crewmates by examining them.
Various factors influence this; 
- The damage dealt by your limbs.
- Your experience in the fitness skill.
- Various quirks, genetic mutations, and brain traumas.

In order to make this not an antagonist check, Changelings will now also be able to copy your athletics skill level when they steal your DNA.
This is... admittedly a little weird, but this is a consequence of using "skills" to represent a physical attribute I guess.

## Why It's Good For The Game

During the social experiment we sometimes call "April Fools Day" it became apparent to me that our playerbase liked to watch a number go up, a quality marked especially in those players who liked to have their characters hang out in the gym.
Many of them made their characters repeatedly hang out in the gym and work out even though doing this had no effect at all on the number they were supposedly increasing.
I would like to give this joy back to them.
<sub><sup>Additionally I have heard that the ability to retain players by stimulating their desire to see numbers increase is good for our annual retention metrics.</sub></sup>

Ultimately this is a very silly feature but I think that there often _can be_ a comparative aspect to bodybuilding and letting players be competitive in terms of looking at each other's completely meaningless numbers gives them something to base a story around.

Also I mostly only make silly features.

## Changelog

:cl:
add: People who spend time improving their bodies are able to evaluate the relative physical strength of those they examine as a kind of "power level".
add: Changeling transformation copies the target's athletics skill level.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/examine.dm [hunk 1]
```diff
@@ -358,6 +358,8 @@
 				msg += "[span_notice("[t_He] appears to have been dissected. Useless for examination... <b><i>for now.</i></b>")]\n"
 			if(HAS_TRAIT(src, TRAIT_SURGICALLY_ANALYZED))
 				msg += "[span_notice("A skilled hand has mapped this one's internal intricacies. It will be far easier to perform future experimentations upon [t_him]. <b><i>Exquisite.</i></b>")]\n"
+		if(HAS_MIND_TRAIT(morbid_weirdo, TRAIT_EXAMINE_FITNESS))
```
Comment (MrMelbert) [inline code/modules/mob/living/carbon/human/examine.dm:361]: `morbid_weirdo` 🤔

**Comment by necromanceranne** [comment]
Just for fun, but if someone is significantly more powerful than you in terms of fitness, can it cause you to become shaken by fear? Maybe just afflicting them with jittering?

File: code/modules/mob/living/carbon/human/human_helpers.dm [hunk 2]
```diff
@@ -328,3 +328,36 @@
...
+		var/obj/item/bodypart/part = get_bodypart(body_zone)
+		if (isnull(part) || part.unarmed_damage_high <= 0 || HAS_TRAIT(part, TRAIT_PARALYSIS))
+			continue
+		min_damage += part.unarmed_damage_low
+		max_damage += part.unarmed_damage_high
+
+	var/damage = ((min_damage / 4) + (max_damage / 4)) / 2 // We expect you to have 4 functional limbs- if you have fewer you're probably not going to be so good at lifting
+
+	var/fitness_level = ceil(damage * (ceil(athletics_level / 2)) * fitness_modifier * 10)
+	return span_notice("You'd estimate [p_their()] fitness level at about [fitness_level].")
```
Comment (Iamgoofball) [inline code/modules/mob/living/carbon/human/human_helpers.dm:363]: gonna have to ask if this can make it up to over 9000

**Comment by EricZilla** [comment]
God I can finally have Lexx say "Lookin huge bro" and actually be able to see that they are in fact looking huge

**Comment by Higgin** [comment]
I need to make a bounty for the weightlifting version of Freezer, Dyel Ngmi.

File: code/modules/mob/living/carbon/human/human_helpers.dm (hunk 2 above)
Comment (Jacquerel) [reply code/modules/mob/living/carbon/human/human_helpers.dm:363]: With the very arbitrary way I set up the calculations, unfortunately I do not think this is a realistically attainable number.
I do like Anne's suggestion though.

**Comment by Zytolg** [comment]
Gotta add gravity training now

**Pull request merged**